app/javascript/App/bundles/common/components/WatchButton.tsx

```diff
@@ -71,4 +72,6 @@ const WatchButton = ({ coin, hasText, loggedIn }: Props) => {
   )
 }
 
+ReactOnRails.register({ WatchButton })
```

Comment:
It makes the component to be available for `react_component` function in rails

Reply:
Is it better to move these `ReactOnRails.register` calls into a single location e.g. the webpack entry point `startup.js`